Revisione activity #2

GiuliaPicasso opened the pull request.

strumet commented:
Ciao Maria Giovanna,
c'è un piccolo errore nella denominazione dell'esercizio EX01 (_892976_ anziché _892967_).
Correggilo e potrò aggiungere i tuoi file al repository del corso.
mf

strumet commented:
:+1:

GiuliaPicasso committed:
Delete 917704.pdf

GiuliaPicasso committed:
Revisione activity #2

Buongiorno Professore,
Ho seguito i suoi consigli e ho aggiunto le viste mancanti alla tavola e corretto eventuali errori confrontandomi con le revisioni già presenti sul sito. Ho comunque avuto due problemi: 
- non sono riuscita ad inserire la linea tratto punto in corrispondenza dell'interruzione del disegno;
- duplicando le viste delle piante per svolgere l'attività in scala 1 a 50, non sono riuscita a rendere le viste "indipendenti" dalla vista da cui le ho copiate (ossia quelle in scala 1 a 100, nonostante le abbia convertite da dipendenti a indipendenti), quindi ogni volta che ne modificavo una, cambiava anche l'altra. Alla fine ho risolto aggiungendo linee sottili dove mancavano.

La ringrazio per l'attenzione

strumet commented:
Ciao Giulia,
la tavola va abbastanza bene. Qua e là sono presenti piccole imperfezioni (doppia linea di sezione, linea tratto-punto per i muri sezionati, murature non correttamente giuntate...) ma non credo che avrai difficoltà a correggerle una volta individuate.

Le difficoltà segnalate [nel tuo messaggio di commit](https://github.com/strumet/strumet.github.io/pull/49/commits/af269c6c06add32e73c163b908104ecb993de9cc) sono relative ad errori compiuti durante la procedura di attuazione dei singoli comandi e pertanto ritengo più utile affrontarle in persona durante le attività di lavoro in classe in modo da poterti seguire nelle varie operazioni.

Qui mi limito quindi a segnalarti l'errore più rilevante presente nella tavola, ovvero, la cancellazione delle pareti relative all'appartamento B: anziché cancellarle è, infatti, necessario escluderle dalla rappresentazione limitando il campo della vista (sia in pianta che in sezione). I comandi per farlo sono quelli relativi al "cropping" della vista: https://knowledge.autodesk.com/support/revit-lt/learn-explore/caas/CloudHelp/cloudhelp/2019/ENU/RevitLT-DocumentPresent/files/GUID-DDD8E523-E1FC-42C5-A89E-488F6DDC5AA4-htm.html

Ultima nota. Piuttosto che inserire i tuoi commenti al momento dei commit (come hai fatto [qui](https://github.com/strumet/strumet.github.io/commit/1b247f4f512265c96a1d8ecc34c9af92b8d08da4), [qui](https://github.com/strumet/strumet.github.io/commit/faca8df286e797e3f069402282886362f7ea8053), e [nel tuo ultimo commit](https://github.com/strumet/strumet.github.io/pull/49/commits/af269c6c06add32e73c163b908104ecb993de9cc)) ti consiglio di utilizzare la finestra del pull request: in questo modo, infatti, i messaggi risultano più evidenti e facili da trovare sia per me che per i tuoi colleghi (infatti non mi ero accorto dei tuoi messaggi precedenti e mi scuso pertanto per non averti risposto). L'uso dei messaggi di commit è un'ottima pratica: ma risulta più utile a tenere traccia del proprio lavoro che a comunicare con gli altri.

Saluti,
mf
